react/components/InputSearch/index.js

```diff
@@ -64,7 +71,9 @@ class InputSearch extends Component {
         onBlur={() => this.handleFocus(false)}
         onMouseEnter={() => this.handleHovering(true)}
         onMouseLeave={() => this.handleHovering(false)}
-        onKeyUp={e => e.key === 'Enter' && this.handleSubmit()}
+        onKeyUp={e =>
+          e.key === 'Enter' && this.handleSubmit(new Event('submit'))
```

Comment:
No risk of the event being `enter` lowercase?

Reply:
In chrome it works like this, tho I did not test in other browsers. :/

Reply:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values#Whitespace_keys), it seems that it's always `Enter`.